Sprint 2

tmj1 opened the pull request:
Sprint 2 Last Version

tmj1 committed:
Merge pull request #1 from tmj1/sprint_1

Sprint 1 Last Version

mikebars1995 reviewed (commented):
К сожалению, работу придется отклонить без потери итерации, так как:



Вам не нужно было закрывать `pull request`, пока работа не будет принята, чтобы видно было историю комментариев и изменений. Нужно присылать всегда одну и ту же ссылку на пулл-реквест каждую итерацию. Как только проектная работа (этап/спринт) будет решена, вот тогда нужно мерджить ветку в `main`.

В пулл-реквесте должен быть код изменных файлов, где я бы мог оставлять комментарии. Вам нужно делать ветку и не трогать ее до тех пор, пока работа не будет принята (решена) после ревью. Только тогда нужно мерджить ветку в `main`. Скрин https://skr.sh/sHCOkTJU6r6

tmj1 committed:
Merge pull request #4 from tmj1/sprint_2

Sprint 2 only for merge

tmj1 committed:
change Readme

tmj1 committed:
Refactoring code

added isEqual function
added screenList
added exports utils
added Screens enum
added Route class
added BrowseRouter class
added props type
added params for another pages
refactoring index

tmj1 committed:
Refactoring code

changes logic of handleSubmitForm function
refactoring BaseAPI class
added AuthAPI class
added signup function
added service for Auth
decomposed code
added getMessageFromResponse function
added close tooltip by click

tmj1 committed:
Refactoring code

added add chat popup element
added formValidator for add chat form
refactoring btnProfile component
added ChatApi
added ChatService
added logic for created of chat
fixed HTTPTransport class
added event click for signout

tmj1 committed:
Refactoring code

added filtered chats
clear code
fixed render error
decomposed code

tmj1 committed:
Refactor code

clear code
added changeUserPassword in ProfileApi
added changeUserPassword in ProfileService
added logic working with change user info
added onClick event for BtnBackProfile component
changes BtnBackProfile component
decomposed code
added base api and service for remove chat logic
addded delete method in BaseAPI
added api and service for find user by login
fixed actions for popup
improvements app
fixed bag

tmj1 committed:
Refactor code

added getChatToken method in ChatApi
added GetChatToken type
added logic taked chat of token
fixed errors
fixed styles

tmj1 committed:
Refactoring

fix errors
refactoring
change logic

tmj1 committed:
Update README.md

tmj1 committed:
Update package.json

tmj1 committed:
Update AuthService.ts

tmj1 committed:
Update AuthService.ts

tmj1 committed:
Update AuthService.ts

tmj1 committed:
Update AuthService.ts

tmj1 committed:
Update ChatService.ts

tmj1 committed:
Update ProfileService.ts

tmj1 committed:
Update AuthService.ts

tmj1 committed:
Update ChatService.ts

tmj1 committed:
Update ProfileService.ts

tmj1 committed:
Update ProfileService.ts

tmj1 committed:
refactor files for test

refactor files for test

tmj1 committed:
Merge branch 'sprint_3' of https://github.com/tmj1/middle.messenger.praktikum.yandex into sprint_3

tmj1 committed:
Update package.json

tmj1 committed:
change links

change links

tmj1 committed:
Merge branch 'sprint_3' of https://github.com/tmj1/middle.messenger.praktikum.yandex into sprint_3

tmj1 committed:
refactoring code for tests

refactoring code for tests

tmj1 committed:
Refactoring code for tests

Refactoring code for tests

tmj1 committed:
Refactor code for tests

Refactor code for tests

tmj1 committed:
Update user.css

tmj1 committed:
Refactoring code for tests

Refactoring code for tests

tmj1 committed:
add mocha

add mocha

tmj1 committed:
Update package.json

tmj1 committed:
Update package.json

tmj1 committed:
Merge branch 'sprint_3' of https://github.com/tmj1/middle.messenger.praktikum.yandex into sprint_3

tmj1 committed:
Update Readme

Update Readme

tmj1 committed:
Update src/components/avatar/avatar.ts

Co-authored-by: Maksim Drobyshev <49103217+maks88sgt@users.noreply.github.com>

tmj1 committed:
Update src/components/user/user.ts

Co-authored-by: Maksim Drobyshev <49103217+maks88sgt@users.noreply.github.com>

tmj1 committed:
Update BrouseRouter.ts

tmj1 committed:
Update MessagesService.ts

tmj1 committed:
Update package.json

tmj1 committed:
Update MessagesService.ts

tmj1 committed:
Update MessagesService.ts

tmj1 committed:
Update MessagesService.ts

tmj1 committed:
refactor

tmj1 committed:
refact

refact

tmj1 committed:
refact

tmj1 committed:
refact

tmj1 committed:
Reformat code

tmj1 committed:
Merge pull request #5 from tmj1/sprint_3

Sprint 3
